src/components/cart-management/cart-management-card/CartManagementCard.js

```diff
@@ -19,6 +21,22 @@ function CartManagementCard({ id, title, author, rating, description, price, pro
             }
         })
     }
+
+    const moveToWishlist = () => {
+        removeItemFromCart();
+        dispatchWishlist({
+            type: "ADD_TO_WISHLIST",
+            payload: {
+                title,
+                author,
+                rating,
+                description,
+                price,
+                productImage,
+                id
+            }
```

Comment:
you can send only product id in payload. It not good practice to share whole product data.